consensus/mint-client/src/config.rs

```diff
@@ -2,14 +2,157 @@
 
 //! Command line configuration for the consensus mint client.
 
-use clap::{Parser, Subcommand};
+use clap::{Args, Parser, Subcommand};
 use hex::FromHex;
 use mc_account_keys::PublicAddress;
 use mc_api::printable::PrintableWrapper;
-use mc_crypto_keys::{DistinguishedEncoding, Ed25519Private};
-use mc_transaction_core::mint::constants::NONCE_LENGTH;
+use mc_crypto_keys::{DistinguishedEncoding, Ed25519Pair, Ed25519Private, Ed25519Public, Signer};
+use mc_crypto_multisig::{MultiSig, SignerSet};
+use mc_transaction_core::mint::{
+    constants::NONCE_LENGTH, MintConfig, MintConfigTx, MintConfigTxPrefix, MintTx, MintTxPrefix,
+};
 use mc_util_uri::ConsensusClientUri;
-use std::{convert::TryFrom, fs};
+use rand::{thread_rng, RngCore};
+use std::{convert::TryFrom, fs, path::PathBuf};
+
+#[derive(Args)]
+pub struct MintConfigTxParams {
+    /// The key(s) to sign the transaction with.
+    #[clap(long = "signing-key", required(true), parse(try_from_str = load_key_from_pem), env = "MC_MINTING_SIGNING_KEYS")]
```

Comment:
Optional: use_value_delimiter to support multiple values -- basically required for env
```suggestion
    #[clap(long = "signing-key", required =true, use_value_delimiter = true, parse(try_from_str = load_key_from_pem), env = "MC_MINTING_SIGNING_KEYS")]
```